src/vs/platform/actions/browser/menuEntryActionViewItem.ts

```diff
@@ -304,14 +308,142 @@ export class SubmenuEntryActionViewItem extends DropdownMenuActionViewItem {
 	}
 }
 
+class DropdownWithDefauktActionViewItem extends BaseActionViewItem {
```

Comment:
Defaukt is probably intended to be Default

Reply:
Indeed - thanks for spotting this.